scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -429,6 +429,11 @@
       (.setBasePath api-client base-path))
     (when (some? verifying-ssl)
       (.setVerifyingSsl api-client verifying-ssl))
+    ; Loading ssl-cert-path must be last SSL operation we do in setting up API Client. API bug.
+    ; See explanation in comments in https://github.com/kubernetes-client/java/pull/200 and issue #199
```

Comment:
Don't think you meant for this to link to

https://github.com/twosigma/Cook/issues/199

Reply:
Its related to both. Technically, we're imitating the same fix as #200, for the bug in #199.

Reply:
Are you sure it's related to:

`https://github.com/twosigma/Cook/pull/199`?

That's what this links to:

https://github.com/twosigma/Cook/pull/199